tests/e2e-integration/integration/datasets/DatasetJSDataverseRepository.spec.ts

```diff
@@ -339,6 +338,7 @@ describe('Dataset JSDataverse Repository', () => {
       expect(dataset.version.title).to.deep.equal(datasetExpected.title)
     })
```

Comment:
We are not testing that the dataset is actually deaccessioned.
I think there is a method to deaccesion a dataset, see DatasetHelper.deaccession method.